content/english/events/20210807.md

```diff
@@ -0,0 +1,40 @@
+---
+title: "Pop: Life in the Middle of Pop Culture"
+date: 2021-08-06T17:31:01+07:00
+slug: "pop-life-in-the-middle-of-pop-culture"
+description: "Pop: Life in the Middle of Pop Culture"
+keywords:
+        [
+                "Fellowship",
+                "Weekly",
+                "Youth",
+                "GRII BSD",
+                "Pemuda",
+                "Christian",
+                "Faith",
+        ]
+tags: ["Fellowship"]
+math: false
+toc: true
+draft: false
+---
+
+![image](/images/events/20210807.jpeg)
+
+Hello, this saturday, our youth fellowship will held a seminar about Pop Culture.
```

Comment:
capitalize some words, in particular change it to this:

`Hello, this Saturday, our Youth Fellowship group will have a Pop Culture seminar. Come and join.

Reply:
done